src/app/patient-dashboard/formentry/previous-encounter-resolver.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable }             from '@angular/core';
```

Comment:
rename service to appropriate name since the work of the service is to fetch the form schema as well as fetch the previous encounter. Probably rename it to 'FormCreationDataResolverService'